### DISCUSSION TITLE
Cloudlog for Field Day

### DISCUSSION THREAD

**Discussion by refutationalist** [magicbug/Cloudlog#1499] [Q&A] [answered]
Hello!

My friends and family want to do ARRL field day this year, so I'm getting set up for it.   Has anyone used Cloudlog for Field Day?   I would presume I'd put the class and section in the exchange field?

I fully admit that I haven't handled much logging before.

**Comment by magicbug** [comment, marked as answer]
Sorry for not replying in time, but I'd suggest not using Cloudlog for FD at the moment.

**Comment by refutationalist** [comment]
Thanks for the update.   I actually went with it for lack of time and managed to massage the needed data with SQL commands.   So it did work!
